### ISSUE TITLE
Output of this module is not normalize which causes the handlebars-helpers test to fail in Windows

### ISSUE THREAD

**Issue by LaurentGoderre** [jonschlinkert/relative#4]

**Comment by jonschlinkert** [comment]
Which handlebars tests? the old ones or the v0.6.0 branch? That's not something this lib should do though, it's better to use a path normalization function to wrap any path method. I'd rather update the handlebars helpers to do that properly in tests.

**Comment by LaurentGoderre** [comment]
The v0.6.0 test. Is it supposed to be for physical paths or URL paths?

**Comment by jonschlinkert** [comment]
file system, so non-URL paths.

**Comment by LaurentGoderre** [comment]
So the test needs to be changed then

**Comment by jonschlinkert** [comment]
yeah, which tests are failing? I use both windows and mac, I'm used to fixing this issue lol

**Comment by unreal666** [comment]
// On Windows.
relative(process.cwd(), './a/b/c.txt');
// '.dev\\a\\b\\c.txt'
